oak-core/src/main/java/org/apache/jackrabbit/oak/query/ast/SelectorImpl.java

```diff
@@ -541,6 +558,21 @@ private boolean nextInternal() {
         return false;
     }
 
+    private void totalQueryStats (QueryEngineSettings queryEngineSettings) {
+        if (updateTotalQueryHistogram) {
+            updateTotalQueryHistogram = false;
+            HistogramStats histogramStats = queryEngineSettings.getStatisticsProvider().getHistogram(SLOW_QUERY_PERCENTILE_METRICS_NAME, StatsOptions.METRICS_ONLY);
+            histogramStats.update(TOTAL_QUERY_HISTOGRAM);
+        }
+    }
+
+    private void addSlowQueryStats(QueryEngineSettings queryEngineSettings){
```

Comment:
(here is no space between ")" and "{"... By the way a simpley way to ensure consistent formatting is to use the IDE auto-format feature.)